### ISSUE TITLE
Update Podcast info at a set interval

### ISSUE THREAD

**Issue by tofumatt** [mozilla/high-fidelity#69]
Podcasts should refresh at least once per day.

We can possibly do this with a background ServiceWorker.

**Comment by mcepl** [comment]
Well, the interval should quite definitively be configurable.

**Comment by tofumatt** [comment]
Yeah, I would like at least an app-wide config option for this :-)


- tofumatt

**Comment by mcepl** [comment]
app-wide should be enough ... at least for the first version, I guess.
